application/src/test/java/org/togetherjava/tjbot/commands/tophelper/TopHelperMessageListenerTest.java

```diff
@@ -46,97 +43,84 @@ static void setUp() {
         MessageReceivedEvent event =
-                createFakeMessageRecievedEvent(true, false, true, OVERVIEW_CHANNEL_PATTERN);
+                createFakeMessageReceivedEvent(true, false, true, OVERVIEW_CHANNEL_PATTERN);
 
         // WHEN checking if the message should be ignored
-        boolean shouldBeIgnored = topHelpersListenerSpy.shouldIgnoreMessage(event);
+        boolean shouldBeIgnored = topHelpersListener.shouldIgnoreMessage(event);
 
         // THEN the message is ignored
         assertTrue(shouldBeIgnored);
     }
 
     @Test
-    @DisplayName("Ignores webhooks")
     void ignoresWebhooks() {
         // GIVEN a message from a webhook
         MessageReceivedEvent event =
-                createFakeMessageRecievedEvent(false, true, true, OVERVIEW_CHANNEL_PATTERN);
+                createFakeMessageReceivedEvent(false, true, true, OVERVIEW_CHANNEL_PATTERN);
 
         // WHEN checking if the message should be ignored
-        boolean shouldBeIgnored = topHelpersListenerSpy.shouldIgnoreMessage(event);
+        boolean shouldBeIgnored = topHelpersListener.shouldIgnoreMessage(event);
 
         // THEN the message is ignored
         assertTrue(shouldBeIgnored);
     }
 
     @Test
-    @DisplayName("Ignores wrong channels")
     void ignoresWrongChannels() {
         // GIVEN a message outside a help thread
         MessageReceivedEvent eventNotAThread =
-                createFakeMessageRecievedEvent(false, false, false, OVERVIEW_CHANNEL_PATTERN);
+                createFakeMessageReceivedEvent(false, false, false, OVERVIEW_CHANNEL_PATTERN);
         MessageReceivedEvent eventWrongParentName =
-                createFakeMessageRecievedEvent(false, false, true, "memes");
+                createFakeMessageReceivedEvent(false, false, true, "memes");
 
         // WHEN checking if the message should be ignored
-        boolean ignoresNonThreadChannels =
-                topHelpersListenerSpy.shouldIgnoreMessage(eventNotAThread);
+        boolean ignoresNonThreadChannels = topHelpersListener.shouldIgnoreMessage(eventNotAThread);
         boolean ignoresWrongParentNames =
-                topHelpersListenerSpy.shouldIgnoreMessage(eventWrongParentName);
+                topHelpersListener.shouldIgnoreMessage(eventWrongParentName);
 
         // THEN the message is ignored
         assertTrue(ignoresNonThreadChannels, "Failed to ignore non-thread channels");
         assertTrue(ignoresWrongParentNames, "Failed to ignore wrong parent channel names");
     }
 
 
-    MessageReceivedEvent createFakeMessageRecievedEvent(boolean isBot, boolean isWebhook,
+    MessageReceivedEvent createFakeMessageReceivedEvent(boolean isBot, boolean isWebhook,
             boolean isThread, String parentChannelName) {
-        Message messageMock = mock(Message.class);
-        ThreadChannelImpl threadMock = mock(ThreadChannelImpl.class, RETURNS_DEEP_STUBS);
+        try (MessageCreateData message = new MessageCreateBuilder().setContent("Test").build()) {
```

Comment:
is it important for the test that it says `"Test"` or would any string work? if latter, use `"any"` to signal that to readers. or, if possible, leave out the `setContent` all together

Reply:
The content is completely unimportant for the check tests

Reply:
The MessageCreateBuilder complains when not setting anything. I will go with "Any"

Reply:
Changed